docs/components/zeebe/technical-concepts/clustering.md

```diff
@@ -42,6 +42,6 @@ This may lead to one node becoming the leader for all partitions. This is not a
 
 ## Commit
 
-Before a new record on a partition can be processed, it must be replicated to a quorum (typically majority) of followers. This procedure is called **commit**. Committing ensures a record is durable, even in case of complete data loss on an individual broker. The exact semantics of committing are defined by the raft protocol.
+Before a new record on a partition can be processed, it must be replicated to a quorum (typically majority) of nodes. This procedure is called **commit**. Committing ensures a record is durable, even in case of complete data loss on an individual broker. The exact semantics of committing are defined by the raft protocol.
```

Comment:
I suggest to use brokers instead of nodes, because broker/s is used in other places in the same document.

Reply:
Makes sense :+1: thanks :)